source/ui/AMToolButton.h

```diff
@@ -7,22 +7,32 @@ class AMToolButton : public QToolButton
 {
 	Q_OBJECT
 
+	Q_PROPERTY(ColorState colorState READ colorState WRITE setColorState NOTIFY colorStateChanged)
```

Comment:
By defining the colorState as a QProperty, we can use it in a CSS-style selector, in AMToolButton.qss.